src/Sulu/Bundle/SecurityBundle/Entity/Role.php

```diff
@@ -54,6 +54,7 @@ public function __construct()
         $this->userRoles = new ArrayCollection();
         $this->groups = new ArrayCollection();
         $this->settings = new ArrayCollection();
+        parent::__construct();
```

Comment:
calls the BaseRole.php which was changed to call the Symfony Role construct to set the `$role` attribute to empty.